stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/channel/internal/ChannelStateLogic.kt

```diff
@@ -335,8 +335,9 @@ internal class ChannelStateLogic(
      * when the user is searching in the channel.
      * @param isNotificationUpdate Whether the message list update is due to a new notification.
      * @param isChannelsStateUpdate Whether the state update comes from querying the channels list.
+     * @param isDatabaseSync Whether the state comes from a database sync.
```

Comment:
```suggestion
```

Reply:
Ops. I believe this comment should be deleted now.